scripts/exwafs_grib2_0p25_blending.sh

```diff
@@ -2,152 +2,69 @@
+    make_NTC_file.pl NOXX10 KKCI "${PDY}${cyc}" NONE "${FIXwafs}/wafs/wafs_blending_0p25_admin_msg" "${COMOUTwmo}/wifs_0p25_admin_msg"
+    make_NTC_file.pl NOXX10 KWBC "${PDY}${cyc}" NONE "${FIXwafs}/wafs/wafs_blending_0p25_admin_msg" "${COMOUTwmo}/iscs_0p25_admin_msg"
+    if [[ "${SENDDBN_NTC}" == "YES" ]]; then
+        "${DBNROOT}/bin/dbn_alert" NTC_LOW WAFS "${job}" "${COMOUTwmo}/wifs_0p25_admin_msg"
+        "${DBNROOT}/bin/dbn_alert" NTC_LOW WAFS "${job}" "${COMOUTwmo}/iscs_0p25_admin_msg"
     fi
 
-    subject="WARNING! Missing UK data for WAFS GRIB2 0P25 blending, ${PDY} t${cyc}z f${fhr} ${job}"
+    subject="WARNING! Missing UK data for WAFS GRIB2 0P25 blending, ${PDY} t${cyc}z ${job}"
     echo "*************************************************************" >mailmsg
     echo "*** WARNING! Missing UK data for WAFS GRIB2 0P25 blending ***" >>mailmsg
     echo "*************************************************************" >>mailmsg
     echo >>mailmsg
     echo "Send alert message to AWC ...... " >>mailmsg
     echo >>mailmsg
-    echo "One or more UK WAFS GRIB2 files not found:" >>mailmsg
-    cat missing_uk_files >>mailmsg
-    echo "Skipping t${cyc}z f${fhr}..." >>mailmsg    
-    cat mailmsg >"${COMOUT}/${RUN}.t${cyc}z.f${fhr}.wafs_blend_0p25_usonly.emailbody"
-    cat "${COMOUT}/${RUN}.t${cyc}z.f${fhr}.wafs_blend_0p25_usonly.emailbody" | mail.py -s "${subject}" "${MAILTO}" -v
-
-    # Distribute US WAFS unblend Data to NCEP FTP Server (WOC) and TOC
-    echo "altering the unblended US WAFS products:"
-    echo " - ${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2"
-    echo " - ${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2.idx"
-
-    if [[ "${SENDDBN}" == "YES" ]]; then
-        "${DBNROOT}/bin/dbn_alert" MODEL WAFS_0P25_UBL_GB2 "${job}" "${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2"
-        "${DBNROOT}/bin/dbn_alert" MODEL WAFS_0P25_UBL_GB2_WIDX "${job}" "${COMINus}/WAFS_0p25_unblended_${PDY}${cyc}f${fhr}.grib2.idx"
-    fi
+    echo "All missing files:" >>mailmsg
+    echo "------------------" >>mailmsg
+    for file in $missing_uk_files ; do
+	cat $file >>mailmsg
+    done
 
-else
-    # Distribute US WAFS unblend Data to NCEP FTP Server (WOC) and TOC
-    if [[ "${SENDCOM}" == "YES" ]]; then
-        cpfs "0p25_blended_${PDY}${cyc}f${fhr}.grib2" "${COMOUT}/WAFS_0p25_blended_${PDY}${cyc}f${fhr}.grib2"
-    fi
-
-    if [[ "${SENDDBN_NTC}" == "YES" ]]; then
-        #   Distribute Data to NCEP FTP Server (WOC) and TOC
-        echo "No WMO header yet"
-    fi
-
-    if [[ "${SENDDBN}" == "YES" ]]; then
-        "${DBNROOT}/bin/dbn_alert" MODEL WAFS_0P25_BL_GB2 "${job}" "${COMOUT}/WAFS_0p25_blended_${PDY}${cyc}f${fhr}.grib2"
-    fi
+    cat mailmsg >"${COMOUT}/${RUN}.t${cyc}z.wafs_blend_0p25_usonly.emailbody"
+    cat "${COMOUT}/${RUN}.t${cyc}z.f${fhr}.wafs_blend_0p25_usonly.emailbody" | mail.py -s "${subject}" "${MAILTO}" -v
```

Comment:
What is the value of `${fhr}` here?  This is not in a for loop over forecast hours.